cgi-bin/LJ/Talk.pm

```diff
@@ -1540,6 +1541,25 @@ sub talkform {
         $basesubject = "Re: $basesubject";
     }
 
+    my $screening = LJ::Talk::screening_level( $journalu, $opts->{ditemid} >> 8 ) || '';
+    $screening = 'A' if $journalu->has_autoscreen($remote);
```

Comment:
nit: This check should be first since it always overwrites the value of whatever `LJ::Talk::screening_level` does. Although now that I look at it, it definitely predates this diff... oh well.

Reply:
Oh huh, and actually I think that's not necessary anymore because I'm duplicating that check down in the `remote.screened` template argument. (In the commit-by-commit version, I couldn't have removed that until all the from fields were dealt with.)